fix: update with jetbrains fixes

loujaybee opened the pull request:
Some misc updates for JetBrains. Main reason for the change is to update the steps relating to installation of the Gateway plugin (and remove previous steps to install the plugin manually).

loujaybee commented on gitpod/docs/ides-and-editors/goland.md:
```diff
@@ -9,13 +9,22 @@ title: GoLand
-1. Navigate to [your preferences page](https://gitpod.io/preferences)
-2. Select "Open In Desktop IDE" and select {title}
-3. Restart any running workspaces
-4. On workspace start, you will now be prompted to open your workspace in a JetBrains IDE
```
This section is expanded on all four supported IDE pages.

loujaybee commented on src/lib/contents/docs/menu.ts:
```diff
@@ -41,20 +41,21 @@ export const MENU: MenuEntry[] = [
     M("JetBrains Gateway", "ides-and-editors/jetbrains-gateway", []),
     M("VS Code Extensions", "ides-and-editors/vscode-extensions"),
     M("Command Line (e.g. Vim)", "ides-and-editors/command-line", []),
-    M("Local Companion", "ides-and-editors/local-companion", [], "beta"),
```
Grouping together IDE's based on release status (beta) rather than alphabetically. 

Open to feedback (CC: @ChristinFrohne / @gtsiolis - as we discussed before in relation to ordering on homepage [[1](https://www.gitpod.io/)])

| Before      | After |
| ----------- | ----------- |
|    ![image](https://user-images.githubusercontent.com/5528307/155321608-10f6787a-22c9-426d-bd65-377d6a37c405.png)   | ![image](https://user-images.githubusercontent.com/5528307/155321550-2f95b4aa-8797-45b7-8945-b4c4cb519789.png)       |

Cahllagerfeld replied:
I'll close #1645 in favor of this I guess.

loujaybee replied:
Ah sorry, @Cahllagerfeld ! I didn't see that before I raised 🙏  [[1](https://github.com/gitpod-io/website/pull/1645#issuecomment-1049092897)]

loujaybee commented on gitpod/docs/troubleshooting.md:
```diff
@@ -21,18 +21,26 @@ The ClearURLs browser extension for [Google Chrome](https://chrome.google.com/we
+
+### Is there a limit of IDE/editor connections to Gitpod?
+
+There is no hard-limit about how many IDE/editors can be connected to a running Gitpod workspace. All connected IDE/editors will have access to the same underlying file system and directories.
```
Just checking there are not any caveats here, @akosyakov ?

akosyakov replied:
It should be alright.

loujaybee commented on gitpod/docs/ides-and-editors/goland.md:
```diff
@@ -9,13 +9,22 @@ title: GoLand
+<figure>
+    <img class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Selecting {title} in the preferences" src="/images/editors/goland.gif">
+    <figcaption>Updating Gitpod IDE preferences to GoLand</figcaption>
+</figure>
```
Does this make most semantic sense? @nisarhassan12 / @mikenikles ? I didn't see other images with captions, not sure if this is the best approach or if there's another... 🙏

nisarhassan12 replied:
Thanks @loujaybee What you did makes the most sense 💯 here.

There are probably other places throughout the site specifically docs and blog where <figure> elements could have been the semantically correct element to use. 

Writing the figure element in markdown is not easy and straightforward. I have updating the Blogging notion [page](https://www.notion.so/gitpod/Blogging-34c2e078569b4174b42784854f42549f) with a section about figures.

loujaybee commented on gitpod/docs/ides-and-editors/index.md:
```diff
@@ -22,22 +22,23 @@ data={{
-- [IntelliJ](ides-and-editors/intellij#desktop)
-- [PhpStorm](ides-and-editors/phpstorm#desktop)
-- [PyCharm](ides-and-editors/pycharm#desktop)
+- [Command Line (e.g. Vim)](ides-and-editors/command-line)
```
Added command line / vim here as an explicit option. Wording on supported IDEs/editors is a little confusing. Maybe we should make more generic "editing code on your gitpod workspace" ... etc.

akosyakov commented:
@loujaybee it does not update `Getting Started: JetBrains Gateway`

After release it should be:
- Install Gateway
- Click `Install Plugin` for Gitpod provider from welcome screen

akosyakov commented on gitpod/docs/ides-and-editors/phpstorm.md:
```diff
@@ -9,13 +9,22 @@ title: PhpStorm
 
-<img class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Selecting PhpStorm in the preferences" src="/images/editors/phpstorm.gif">
+1. **Install the [latest “release” version](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) of JetBrains Gateway** - With [JetBrains Gateway and Gitpod](jetbrains-gateway) you can create and manage your latest 20 Gitpod workspaces.
+2. **Install the Gitpod plugin** - Navigate to the JetBrains Gateway "Remote Development" page, and click "install" to install the [Gitpod plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway) (no JetBrains Gateway restart is required).
```
(minor) page -> wizard

from [JB docs](https://www.jetbrains.com/help/idea/remote-development-a.html#launch_gateway):
> In the Remote Development wizard, click Connect via SSH.

loujaybee commented:
> @loujaybee it does not update Getting Started: JetBrains Gateway - @akosyakov 

Updated: e9f286e4162a4c1f4eacda4654d28ce96dc0c38b — this is the same instructions copy as the other JetBrains pages 🙏

> We should also mention that the plugin is not auto updated, so in case of troubles make sure to update to the latest plugin from plugin preferences.

Updated: bf16aa16581bcc929aa55870975d9defc472371b

Added a troubleshooting question, let's also add to the other pages if required 🙏

felladrin commented on gitpod/docs/ides-and-editors/pycharm.md:
```diff
@@ -9,13 +9,22 @@ title: PyCharm
 # {title}
 
-> **Please note:** JetBrains PyCharm Desktop is currently in beta. If you experience any issues, or have feedback, please [get in touch](https://www.gitpod.io/support).
+Gitpod enables code editing using JetBrains PyCharm currently via [JetBrains Gateway](jetbrains-gateway).
```
```suggestion
Gitpod enables code editing using JetBrains {title} currently via [JetBrains Gateway](jetbrains-gateway).
```

loujaybee replied:
Thanks @felladrin, updated in 6641e7bbba2d9cbd8190c269690b40251243a69d

felladrin reviewed (commented):
Looks good to me! Left just a few suggestions.

akosyakov commented:
Let's hold this PR for now. New GW is required for it.

akosyakov commented:
@loujaybee since new GW is go out only end of April, maybe we extract from this PR useful bits to another PR?

loujaybee commented on gitpod/docs/ides-and-editors/jetbrains-gateway.md:
```diff
@@ -11,17 +11,20 @@ title: JetBrains Gateway
 
-1. Create a new Gitpod workspace
-2. View 20 latest running Gitpod workspaces
+1. Create a new Gitpod workspace based on
```
Oh wow, good catch — no that's not correct 😆 let me update!

loujaybee replied:
Fixed in: 69a2b5dacfa1edabef7c3b01d3f534b1a104c1b4

loujaybee commented on gitpod/docs/ides-and-editors/intellij.md:
```diff
@@ -9,13 +9,22 @@ title: IntelliJ
+1. **Install the [latest “release” version](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) of JetBrains Gateway** - With [JetBrains Gateway and Gitpod](jetbrains-gateway) you can create and manage your latest 20 Gitpod workspaces.
+2. **Install the Gitpod plugin** - Navigate to the JetBrains Gateway "Remote Development" wizard, and click "install" to install the [Gitpod plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway) (no JetBrains Gateway restart is required).
+3. **Update your Gitpod preferences** - Select IntelliJ on the [Gitpod preferences page](https://gitpod.io/preferences) which will set IntelliJ as your default Desktop IDE for future workspace starts.
+4. **Start (or restart) your workspace** - Either start a workspace directly from within the [JetBrains Gateway](jetbrains-gateway) via the Gitpod plugin **OR** open a new workspace direclty in Gitpod where on workspace start you will be prompted to open IntelliJ for that workspace.
```
Yep, good shout! I agree, updated in: 69a2b5dacfa1edabef7c3b01d3f534b1a104c1b4

loujaybee commented on gitpod/docs/ides-and-editors/intellij.md:
```diff
@@ -9,13 +9,22 @@ title: IntelliJ
 # {title}
 
-> **Please note:** JetBrains {title} Desktop is currently in beta. If you experience any issues, or have feedback, please [get in touch](https://www.gitpod.io/support).
+Gitpod enables code editing using JetBrains IntelliJ currently via [JetBrains Gateway](jetbrains-gateway).
```
Thanks @felladrin, updated in 6641e7bbba2d9cbd8190c269690b40251243a69d

loujaybee commented on gitpod/docs/ides-and-editors/phpstorm.md:
```diff
@@ -9,13 +9,22 @@ title: PhpStorm
 # {title}
 
-> **Please note:** JetBrains {title} Desktop is currently in beta. If you experience any issues, or have feedback, please [get in touch](https://www.gitpod.io/support).
+Gitpod enables code editing using JetBrains PhpStorm currently via [JetBrains Gateway](jetbrains-gateway).
```
Thanks @felladrin, updated in 6641e7bbba2d9cbd8190c269690b40251243a69d

akosyakov commented on gitpod/docs/ides-and-editors/goland.md:
```diff
@@ -9,13 +9,20 @@ title: GoLand
-### Getting Start with {title} Desktop in Gitpod
+### Getting Started
+
+1. **Install the [latest “release” version](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) of JetBrains Gateway** - With [JetBrains Gateway and Gitpod](jetbrains-gateway) you can create and manage your latest 20 Gitpod workspaces.
```
as for now latest release is not requirement anymore, EAP works too

loujaybee replied:
That makes life easier! Updated in: b83270fc2138ff8cd3a4dde49a2c4d2568b6bd5b

akosyakov commented on gitpod/docs/ides-and-editors/goland.md:
```diff
@@ -9,13 +9,20 @@ title: GoLand
+### Getting Started
+
+1. **Install the [latest “release” version](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) of JetBrains Gateway** - With [JetBrains Gateway and Gitpod](jetbrains-gateway) you can create and manage your latest 20 Gitpod workspaces.
+2. **Install the Gitpod plugin** - Navigate to the JetBrains Gateway preferences page and select the plugins tab to search and install the [Gitpod plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway) (no JetBrains Gateway restart is required).
```
Do we need to mention `(no JetBrains Gateway restart is required)` if it just works?

loujaybee replied:
True, removed in b83270fc2138ff8cd3a4dde49a2c4d2568b6bd5b

akosyakov commented on gitpod/docs/ides-and-editors/goland.md:
```diff
@@ -9,13 +9,20 @@ title: GoLand
+</figure>
 
-<img class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Selecting {title} in the preferences" src="/images/editors/goland.gif">
+**Important:** The early access programs (EAP) version of JetBrains Gateway isn’t yet compatible with the [Gitpod Gateway plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway), so please ensure that you download the “release” / stable version, not the EAP version. You must also stop/restart any started workspaces after selecting an IDE preferences for your IDE preferences to take eff ...
```
not true anymore

loujaybee replied:
Reduced copy text in: b83270fc2138ff8cd3a4dde49a2c4d2568b6bd5b

akosyakov commented on gitpod/docs/troubleshooting.md:
```diff
@@ -22,18 +22,41 @@ The ClearURLs browser extension for [Google Chrome](https://chrome.google.com/we
+
+If you are having issues opening your Gitpod workspace via JetBrains IDE:
+
+1. Ensure that you are using the latest ["release" version of JetBrains Gateway](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) (Not EAP)
```
we support EAP version, but generally recommending to try with stable version is still good

loujaybee replied:
Agree, tweaked to remove EAP reference in b83270fc2138ff8cd3a4dde49a2c4d2568b6bd5b

akosyakov reviewed (approved):
LGTM

loujaybee commented:
Thanks @akosyakov ! 🙏 🛹 🚢
